.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "site-shared"]
```

Comment:
I had a long comment here about different ways to name this repo, but I think `site-shared` is the best possible name.